sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/AnalyzeTextsImpl.java

```diff
@@ -42,7 +43,7 @@ public final class AnalyzeTextsImpl {
      *
      * @param client the instance of the service client containing this operation class.
      */
-    AnalyzeTextsImpl(MicrosoftCognitiveLanguageServiceTextAnalysisImpl client) {
```

Comment:
revert?

Reply:
It would need to make it public so other package can use it internally